arm-sql/2014-04-01/swagger/connectionPolicies.json

```diff
@@ -0,0 +1,275 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Server API spec",
+    "description": "The Azure SQL Server management API provides a RESTful set of web services that interact with Azure SQL Server services to manage your databases. The API enables users update server connection policy.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/connectionPolicies/{connectionPolicyName}": {
+      "put": {
+        "tags": [
+          "ConnectionPolicies"
+        ],
+        "operationId": "Servers_CreateOrUpdateConnectionPolicy",
+        "description": "Creates or updates the server's connection policy.",
+        "x-ms-examples": {
+          "Create or update a server's secure connection policy": { "$ref": "../examples/ServerConnectionPolicyCreateOrUpdate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "connectionPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": ["default"],
+            "description": "The name of the connection policy.",
+            "x-ms-enum": {
+              "modelAsString": true,
+              "name": "ConnectionPolicyName"
+            }
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerConnectionPolicy"
+            },
+            "description": "The required parameters for updating a secure connection policy."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerConnectionPolicy"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ServerConnectionPolicy"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
```

Comment:
Noticed that the tag in the tags section for get and put are different.  Shouldn't they be the same?